### PR TITLE
tests: [fix] retry monitoring/exporters test

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by sameersbn** [comment]
first attempt of fixing this was a fail: https://github.com/bitnami/kube-prod-runtime/pull/502

File: tests/monitoring_test.go [hunk 1]
```diff
@@ -92,40 +92,29 @@ var _ = Describe("Exporters", func() {
 	})
 
 	DescribeTable("Exported timeseries", func(selector string, match types.GomegaMatcher) {
+		var series []map[string]string
```
Comment (anguslees) [inline tests/monitoring_test.go]: I think this should be inside `Eventually(func() { ... })` too..

File: tests/monitoring_test.go [hunk 2]
```diff
@@ -92,40 +92,29 @@ var _ = Describe("Exporters", func() {
...
+				Services("kubeprod").
+				ProxyGet("http", "prometheus", "9090", "api/v1/series", params).
+				DoRaw()
+			Expect(err).NotTo(HaveOccurred())
+
+			resp := promResponse{}
+			json.Unmarshal(resultRaw, &resp)
+			json.Unmarshal(resp.Data, &series)
+
+			fmt.Printf("%v\n", series)
```
Comment (anguslees) [inline tests/monitoring_test.go]: Don't write to stdout, write to `GinkgoWriter`.  That way the logging output is correctly associated with the test, and appears only if `ginkgo -v` is used.

File: tests/monitoring_test.go [hunk 3]
```diff
@@ -92,40 +92,29 @@ var _ = Describe("Exporters", func() {
...
+
+			return series, err
+		}, "5m", "5s").Should(match)
 	},
-		Entry("prometheus", `prometheus_tsdb_head_chunks{kubernetes_namespace="kubeprod",name="prometheus"}`, Not(BeEmpty())),
+		Entry("prometheus", `prometheus_tsdb_head_chunks{kubernetes_namespace="kubeprod",name="none"}`, Not(BeEmpty())),
```
Comment (anguslees) [inline tests/monitoring_test.go]: Is this necessary? `name="none"` seems like it might be a bug somewhere.

File: tests/monitoring_test.go (hunk 1 above)
Comment (sameersbn) [reply tests/monitoring_test.go]: done!

File: tests/monitoring_test.go (hunk 2 above)
Comment (sameersbn) [reply tests/monitoring_test.go]: oops! this was not supposed to be committed

File: tests/monitoring_test.go (hunk 3 above)
Comment (sameersbn) [reply tests/monitoring_test.go]: damn! should not have been there.

**Comment by sameersbn** [comment]
@anguslees PTAL!

**Pull request merged**